examples/apps/heart_disease_eval/client/heart_gui.py

```diff
@@ -323,15 +323,16 @@ def evaluate(self, message):
 
         # Set text for JSON sidebar
         req_id = 51
-        self.request_json = wo_params.to_string(req_id)
+        self.request_json = wo_params.to_jrpc_string(req_id)
 
-        work_order_instance = direct_jrpc.create_work_order(
+        work_order_instance = direct_jrpc.get_work_order_instance(
             config
```

Comment:
Do we need to pass config parameter in this function ? We can pass config in the constructor of direct_jrpc.

Reply:
Not required